packages/backend/src/shared/pagination.test.ts

```diff
@@ -58,6 +59,52 @@ describe('Pagination', (): void => {
     )
   })
   describe('getPageInfo', (): void => {
+    describe('payment pointers', (): void => {
```

Comment:
I'm going to have a similar comment that I left on @BlairCurrey 's PR:
- https://github.com/interledger/rafiki/pull/1454#discussion_r1204616058

I think we should have pagination tests for payment pointers live in the `PaymentPointerService` tests

Reply:
Happy to change that. I also found it weird that it was where it was but I put it there for consistency.